catalog/Game_Development.yml

```
@@ -45,7 +45,7 @@ shards:
   mirrors:
   - github: neutrinog/prism
     role: legacy
-- github: AregevDev/raylib-cr
+- github: sol-vin/raylib-cr
   description: Crystal wrapper around raylib
```

We don't want to break exiting references to the legacy shard. When updating the canonical location, a reference to the previous one should be retained with `legacy` role (see https://github.com/shardbox/catalog#how-to-update-a-shards-location).

```suggestion
- github: sol-vin/raylib-cr
  description: Crystal wrapper around raylib
  mirrors:
  - github: AregevDev/raylib-cr
    role: legacy
```